modules/market/market.models.ts

```diff
@@ -8,3 +8,112 @@ export const Market = mongo.createModel<Types.MarketDocument>('Market', {
 export const MarketPair = mongo.createModel<Types.MarketPairDocument>('MarketPair', {
   value: { type: String, required: true },
 });
+
+export const MarketExchange = mongo.createModel<Types.MarketExchangeDocument>('MarketExchange', {
+  value: { type: String },
```

Comment:
not needed? delete from others if not needed

Reply:
i found this one there already

i should remove the two model?